telemetry/converter.go

```diff
@@ -0,0 +1,16 @@
+package telemetry
+
+import "github.com/grafana/grafana-plugin-sdk-go/data"
+
+// Converter can convert input to Grafana Data Frames.
+type Converter interface {
+	Convert(data []byte) ([]FrameWrapper, error)
+}
+
+// FrameWrapper is a wrapper over data.Frame.
+type FrameWrapper interface {
```

Comment:
fine for now... but maybe we should add "key" as a standard frame property :thinking:

Reply:
hmm, maybe not - because the key here used mostly to construct a channel name to publish data into, maybe better try to keep live specifics separate from general data frame logic as long as possible?